caddyhttp/markdown/setup.go

```diff
@@ -39,10 +41,30 @@ func setup(c *caddy.Controller) error {
 
 	cfg := httpserver.GetConfig(c)
 
+	// Add markdown index files to *SiteConfig.IndexPages
+	for _, mdc := range mdconfigs {
+		for _, indexFile := range mdc.IndexFiles {
+			inIndexPages := false
+			for _, indexPage := range cfg.IndexPages {
+				if indexFile == indexPage {
+					inIndexPages = true
+				}
+			}
+			if inIndexPages == false {
+				cfg.IndexPages = append(cfg.IndexPages, indexFile)
+			}
+		}
+	}
+
```

Comment:
@mholt I think you were suggesting doing something like this. This code gets the correct result into cfg.IndexPages without duplicates, but being a bit new to Go, I'm not sure this is the best way as it looks a bit messy.

Reply:
That works! It's clear and easy to read.